Tests/RxCocoaTests/RXObjCRuntime+Testing.m

```diff
@@ -307,7 +307,7 @@ -(NSInteger)message_allSupportedParameters:(id __nullable)p1
                                        p16:(some_insanely_large_struct_t)p16 {                                                                 \
     [self.privateBaseMessages addObject:A(                                                                                                     \
         p1 ?: [NSNull null],                                                                                                                   \
-        p2 ?: [NSNull null],                                                                                                                   \
+        (id)p2 ?: [NSNull null],                                                                                                               \
```

Comment:
Without these changes, the library built successfully, but Xcode could not build the tests with the following error:
```
.../RxSwift/Tests/RxCocoaTests/RXObjCRuntime+Testing.m:604:1: error: incompatible operand types ('Class _Nullable' and 'NSNull * _Nonnull') [-Werror]
IMPLEMENT_OBSERVING_CLASS_PAIR_FOR_TEST(shared)
^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
.../RxSwift/Tests/RxCocoaTests/RXObjCRuntime+Testing.m:56:59: note: expanded from macro 'IMPLEMENT_OBSERVING_CLASS_PAIR_FOR_TEST'
#define IMPLEMENT_OBSERVING_CLASS_PAIR_FOR_TEST(testName) _IMPLEMENT_OBSERVING_CLASS_PAIR_FOR_TEST(testName,,)
                                                          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
.../RxSwift/Tests/RxCocoaTests/RXObjCRuntime+Testing.m:310:12: note: expanded from macro '_IMPLEMENT_OBSERVING_CLASS_PAIR_FOR_TEST'
        p2 ?: [NSNull null],                                                                                                                   \
        ~~ ^  ~~~~~~~~~~~~~
.../RxSwift/Tests/RxCocoaTests/RXObjCRuntime+Testing.m:17:49: note: expanded from macro 'A'
#define A(...) [Arguments argumentsWithValues:@[__VA_ARGS__]]
                                                ^~~~~~~~~~~
```
It was complaining that `p2` being a `Class` does not go well along with a `NSNull` so I made sure `p2` was seen as an `id` with a cast.